### PR TITLE
Initial setup of posting images

### PR DESCRIPTION
**Pull request by mark-drastrup**
Getting the image from the phone, but post request fails

### CODE REVIEW

**Comment by mark-drastrup** [comment]
I have just commited a component to update the profile.

File: components/EditProfileModal.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    textAlign: 'center',
+    color: '#fff',
+    fontWeight: 'bold',
+    fontSize: 18
+  }
+})
+
+const mapStateToProps = state => {
+  return {
+    editProfile: state.editProfile,
```
Comment (bkhartmire) [inline components/EditProfileModal.tsx]: showEditProfileForm

File: components/NewEvent.tsx [hunk 2]
```diff
@@ -60,6 +65,39 @@ class NewEvent extends React.Component<Props, State> {
   }
   
 
+  getPermission = async () => {
+    if (Constants.platform.ios) {
+      const { status } = await Permissions.askAsync(Permissions.CAMERA_ROLL);
+      if (status !== 'granted') alert('Sorry, we need camera roll permissions to make this work!');
+      else this.pickImage()
+    }
+    console.log("This is an Android device #FuckYeah")
```
Comment (bkhartmire) [inline components/NewEvent.tsx]: 100¥

File: components/EditProfileModal.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+interface State {
+  first_name: string,
+  last_name: string,
+  email: string,
+  phone_number: string,
+  profile_photo: string,
+  interests: Array<string>
+}
+
+class EditProfileModal extends Component<Props, State> {
```
Comment (bkhartmire) [inline components/EditProfileModal.tsx]: rename to EditProfile for consistency?

File: src/reducers/reducer.js [hunk 4]
```diff
@@ -9,6 +9,7 @@ const initialState = {
   allEvents: [],
   chats: [],
   showProfile: false,
+  editProfile: false,
```
Comment (bkhartmire) [inline src/reducers/reducer.js]: showEditProfileForm

**Pull request merged**